enumflags/src/fallible.rs

```diff
@@ -0,0 +1,27 @@
+use core::convert::TryFrom;
+use super::{BitFlags, FromBitsError};
+use super::_internal::RawBitFlags;
+
+macro_rules! impl_try_from {
+    () => { };
+    ($ty:ty, $($tt:tt)*) => {
+        impl_try_from! { $ty }
+        impl_try_from! { $($tt)* }
+    };
+    ($ty:ty) => {
```

Comment:
It's not necessary to use 3 rules here. I've pushed a commit onto your branch that shows how this can be done, though ideally it would be squashed into this one.